src/modules/dirwatcher.js

```diff
@@ -0,0 +1,140 @@
+import fs from 'fs';
+import AppEmitter from './../AppEmitter.js';
+
+export default class DirWatcher {
+  constructor() {
+    this.eventEmitter = new AppEmitter();
+  }
+
+  getEventEmitter() {
+    return this.eventEmitter;
+  }
+
+  watch (path, delay) {
+    let previousStats = [];
+
+    setInterval(() => {

```

Comment:
Do you clear this interval?

Reply:
Please, could you show me where I should do that? I don't have method for rejecting watcher, so I don't see place for clearing...